packages/core/core.ts

```diff
@@ -2,11 +2,8 @@ import { ComponentType, StatelessComponent, createElement } from 'react'
 import { cn, NoStrictEntityMods, ClassNameFormatter } from '@bem-react/classname'
 import { classnames } from '@bem-react/classnames'
 
-function getDisplayName<T>(Component: ComponentType<T> | string) {
-  return typeof Component === 'string'
-    ? Component
-    : Component.displayName || Component.name || 'Component'
-}
+const getDisplayName = (Component: ComponentType | string) =>
```

Comment:
Why u rewrite for function expression? All code have function declaration

Reply:
Agree, will make it back to FD